api/client/info.go

```diff
@@ -76,13 +76,13 @@ func (cli *DockerCli) CmdInfo(args ...string) error {
 
 	fmt.Fprintf(cli.out, "Debug mode (client): %v\n", utils.IsDebugEnabled())
 	fmt.Fprintf(cli.out, "Debug mode (server): %v\n", info.Debug)
+	fmt.Fprintf(cli.out, "Docker Root Dir: %s\n", info.DockerRootDir)
```

Comment:
This is not the right place since the format is basically is debug enabled, then the debug items are indented.
So maybe above the client debug value?